### PR TITLE
Lrdocs 4404 theming portlets

### PR DESCRIPTION
**Pull request by nealrobbins**
Merged conflicts and expanded intro.  Still have questions about 01-theming-portlets and 03-embedding-portlets-in-themes-and-layout-templates--@mwilliams2014 can you take a look at those?

### CODE REVIEW

**Comment by nealrobbins** [comment]
Supercedes #2830

File: develop/tutorials/articles/14-themes-and-layout-templates/03-portlets-and-themes/01-theming-portlets.markdown [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
 This tutorial demonstrates how to style portlets with your themes.
 
 ## Portlet FTL [](id=portlet-ftl)
-<!--It doesn't seem to me that this section
-is giving instructions on how to do anything. I'm told that descriptive texts
-of this sort belong in /reference, not /tutorials-NR-->
-Here is a quick look at the default 
+
+Although you can individually style a portlet via the theme's CSS or the
+portlet's *Look and Feel Configuration* menu,<!--Can we link to docs on these
```
Comment (mwilliams2014) [inline develop/tutorials/articles/14-themes-and-layout-templates/03-portlets-and-themes/01-theming-portlets.markdown]: The Look and Feel configuration menu will be in the User Guide docs once they're published. I can link to those. To style a portlet's CSS, the user would have to overwrite the default styles provided by the classes defined in the portlet.ftl file. We don't have any docs on how to do that. That would be up to the dev to figure out the specific styling. I don't think we can do more than saying what we do: 

> To provide a custom style for all portlets, use the CSS classes in this file for the various container elements...

Comment (nealrobbins) [reply develop/tutorials/articles/14-themes-and-layout-templates/03-portlets-and-themes/01-theming-portlets.markdown]: Ok. Thanks.

**Comment by sez11a** [comment]
LRDOCS-4403
LRDOCS-4404
LRDOCS-4405
LRDOCS-4406

**Comment by sez11a** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**